README.md

```diff
@@ -2378,7 +2378,18 @@ Seriously, try the Heroku demo again:
 Feel that buttery-smooth page transition.
 
 
-<br />
+### 12 (Bonus!) Adding authentication
+
+Currently, the application allows *anyone* 
+to access it and manage todo items.
+However, wouldn't it be awesome if 
+we added *authentication* so each user
+could check their own list?
+
+We created a [small guide](./auth.md)
```

Comment:
"small guide" is not a very informative link text. see: https://github.com/dwyl/contributing/issues/119
We're better off with just the file name:
```suggestion
We created a dedicated authentication guide: 
[`/auth.md`](./auth.md)
```